src/header/header.scss

```diff
@@ -265,4 +292,51 @@
       }
     }
   }
+
+  .iui-button-split-menu {
+    height: 100%;
+    padding: 0 0 0 $iui-xs;
+    display: flex;
+    overflow: hidden;
+    max-width: 25vw;
+
+    .iui-button {
+      padding: 0 $iui-xs;
+    }
+
+    &.iui-active {
+      // Basic active styling - blue background and bottom border
+      @include themed {
+        box-shadow: 0 2px 0 0 t(iui-color-foreground-primary);
+      }
+
+      &,
+      &:hover {
+        @include themed {
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          );
+        }
+      }
+
+      // Removing button hovers and backgrounds and adding last child hover
+      .iui-button {
+        background-color: transparent !important;
+
+        &::after {
+          background-color: transparent;
+        }
+      }
+
+      .iui-button:last-child:hover {
+        @include themed {
+          background-color: rgba(
+            t(iui-color-foreground-primary-rgb),
+            t(iui-opacity-6)
+          ) !important;
```

Comment:
Lets try to remove these `!important`s.

What rule do you need to override and what is the specificity of it? We should be able to boost the specificity in header (at least temporarily).

Reply:
iui-active has at least 4 different background-color rules 
![image](https://user-images.githubusercontent.com/83585998/128360247-51fa0998-a07f-4617-aeee-cd24d511f2ad.png)

!important overrides all of them. This should be fixed when reduce specificity task is completed.

Reply:
looks like a specificity of 0-6-0 for active state, and 0-7-0 for active hover state.

So we need at least 0-6-1 and 0-7-1 respectively. Right now those two rules have 0-4-0 and 0-6-0.

One easy thing you can do is add a modifier on `.iui-button-split-menu`.
Another thing you can do is add `.iui-borderless` to the `.iui-button` selectors.
And then, you can also add `button` to boost the last part.

Reply:
Removed !important rules by adding borderless selector.

Reply:
That's great!

Is `::after` `one really needed? If yes, you could combine with the other selectors above.

Also I still think we should have a modifier class on `.iui-button-split-menu` (call it `.iui-header-split-menu` or something). Right now our selector is tackling all split buttons at all levels inside the header, and it could cause unnecessary problems.

Reply:
::after is needed to remove top border. Changed styling to display: none
Added iui-header-split-menu modifier class